assets/panel/debugger.properties

```diff
@@ -677,14 +677,17 @@ functionSearchSeparatorLabel=←
 # text displayed when the user searches for specific lines in a file
 gotoLineModal.placeholder=Go to line…
 gotoLineModal.key=CmdOrCtrl+Shift+;
+gotoLineModal.title=Go to a line number in a file
```

Comment:
Good call on the translations! 😄 It might be nice at some point to unify all the translations under `quickOpenModal` rather than split across `gotoLineModal`, `symbolSearch`, etc.